environment/galayou-y4-t31l-atbm6012bx.txt

```diff
@@ -17,8 +17,8 @@ gpio_mmc_cd=50
 gpio_motor_h=49 61 62 63
 gpio_motor_v=52 59 64 53
 gpio_speaker=7o
-gpio_led_w=10O
-gpio_wlan=17o
+gpio_white=10O
+gpio_wlan=17
```

Comment:
the suffix is needed to know if the wlan device turns on, if the gpio is low, or high.

Reply:
Thnaks. I will change that.